packages/node-utils/src/edge-to-node/headers.ts

```diff
@@ -0,0 +1,109 @@
+import type { Headers } from '@edge-runtime/primitives'
+import type { OutgoingHttpHeaders, ServerResponse } from 'node:http'
+
+export function toOutgoingHeaders(
+  headers?: Headers & { raw?: () => Record<string, string> }
+): OutgoingHttpHeaders {
+  const outputHeaders: OutgoingHttpHeaders = {}
+  if (headers) {
+    for (const [name, value] of typeof headers.raw !== 'undefined'
+      ? Object.entries(headers.raw())
+      : headers.entries()) {
+      outputHeaders[name] = value
+      if (name.toLowerCase() === 'set-cookie') {
+        outputHeaders[name] =
+          'getAll' in headers
+            ? headers.getAll('set-cookie')
+            : splitCookiesString(value)
```

Comment:
Is this line reachable? I wonder if `splitCookiesString` is covered in tests.

Reply:
It's a legit question. Is is covered, by this test:
https://github.com/vercel/edge-runtime/blob/17ff93207699a6dfda2c0c9ab05ea69619bdb32a/packages/node-utils/test/edge-to-node/headers.test.ts#L27-L41